publication-commons/src/main/resources/features/publish_publication.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Publish a Publication
+
+  Scenario: A User without required permissions tries to publish a Publication
+    Given a User wants to publish a Publication
+    But they are missing the required permissions
```

Comment:
If And/But are interchangeable, then I guess this is fine (I have never used it, and the documentation doesn't explicitly mention this usage)